### PR TITLE
Function to normal order in chemistry convention

### PR DESCRIPTION
**Pull request by babbush**
This will be used for the low rank stuff. @kevinsung are my doc strings okay?

### CODE REVIEW

**Comment by babbush** [comment]
@kevinsung I've added a test that absolutely checks for compliance with chemist ordering. I was asking about the ":math:" thing in the doc strings. Does that look okay / will it render okay in sphinx? If so, I think we're now good to merge.

File: src/openfermion/utils/_operator_utils.py [hunk 1]
```diff
@@ -37,6 +37,54 @@ class OperatorUtilsError(Exception):
...
+class OperatorSpecificationError(Exception):
+    pass
+
+
+def chemist_ordered(fermion_operator):
+    """Puts a two-body fermion operator in chemist ordering.
+
+    The normal ordering convention for chemists is different.
+    Rather than ordering the two-body term as physicists do, as
+    :math: a^\dagger a^\dagger a a
```
Comment (kevinsung) [inline src/openfermion/utils/_operator_utils.py]: It should be :math:`a^\dagger a^\dagger a a`

Comment (kevinsung) [reply src/openfermion/utils/_operator_utils.py]: I mean,
```
:math:`a^\dagger a^\dagger a a`
```

File: src/openfermion/utils/_operator_utils.py [hunk 2]
```diff
@@ -46,9 +46,9 @@ def chemist_ordered(fermion_operator):
 
     The normal ordering convention for chemists is different.
     Rather than ordering the two-body term as physicists do, as
-    :math: a^\dagger a^\dagger a a
+    :math: `a^\dagger a^\dagger a a`
```
Comment (kevinsung) [inline src/openfermion/utils/_operator_utils.py]: No space between :math: and the backtick.

**Pull request merged**